src/subscribe.rs

```diff
@@ -36,7 +36,7 @@ use tracing::{debug, error, info, warn};
 
 use crate::client::{self, Certs, SERVER_RETRY_INTERVAL};
 
-const REQUIRED_GIGANTO_VERSION: &str = "0.23.0";
+const REQUIRED_GIGANTO_VERSION: &str = "0.24.0";
```

Comment:
Since `REQUIRED_GIGANTO_VERSION` follows the version of Data-Store, there is no need to modify it at this time. Therefore, updates to `REQUIRED_GIGANTO_VERSION` will be made later when compatibility work with Data-Store is required.